usagereportsv4/usage_reports_v4_examples_test.go

```diff
@@ -167,6 +167,28 @@ var _ = Describe(`UsageReportsV4 Examples Tests`, func() {
 			Expect(response.StatusCode).To(Equal(200))
 			Expect(accountUsage).ToNot(BeNil())
 		})
+		It(`GetAccountUsage request example with names`, func() {
```

Comment:
We can really only accommodate one request example per operation.  I think you could either incorporate the new "names" param in the existing example for this operation above, or perhaps just avoid demonstrating the "names" param altogether.   It's up to you how to handle.
This comment also applies to the other situations where you added a second request example for an operation below.

Reply:
Updated. Thanks!